### PR TITLE
updates for seperate waveforms

### PR DESCRIPTION
**Pull request by petergaemers**
Before you submit this PR: make sure to put all operations-related information in a wiki-note, a PR should be about code and is publicly accessible

**What is the problem / what does the code in this PR do**
This PR add the top and bottom array waveforms to peaklets and beyond. For this small changes to the arguments of some peak_building/splitting functions where needed. Do not merge before, or much later than https://github.com/AxFoundation/strax/pull/393

**Can you briefly describe how it works?**
See https://github.com/AxFoundation/strax/pull/393
**Can you give a minimal working example (or illustrate with a figure)?**
`st.make(run_id,'peaks')

Waveforms

Ps tests fail untill the strax pr is merged

### CODE REVIEW

File: straxen/plugins/peaklet_processing.py [hunk 1]
```diff
@@ -499,6 +502,8 @@ def compute(self, peaklets_he):
 
 @export
 @strax.takes_config(
+    strax.Option('n_top_pmts', type=int,
+                 help='Number of PMTs in the top array'),
```
Comment (WenzDaniel) [inline straxen/plugins/peaklet_processing.py:506]: Have you changed the option here by accident? At least the plugin itself stayed the same. Or I am blind, not unlikely on a Friday afternoon :D

**Review by WenzDaniel — COMMENTED** [review]
Hej, Peter thanks for this awesome work. Let me also check strax before I approve.

File: straxen/plugins/peaklet_processing.py (hunk 1 above)
Comment (petergaemers) [reply straxen/plugins/peaklet_processing.py:506]: I think you're confusing n_TOP_pmts with n_TPC_pmts

Comment (petergaemers) [reply straxen/plugins/peaklet_processing.py:506]: That was a mistake finding why merging crashed

**Pull request closed without merging**